### ISSUE TITLE
 Interface\AddOns\Carbonite.Quests\NxQuest.lua:4704: assertion failed!

### ISSUE THREAD

**Issue by Thrumbar** [mikepauer/Carbonite#653]
Get this if I click on quest in windows to track while in combat.  The quest is a Scenario in an instance for Stormwind Stockade..  The error doen not appear if I click on any other quest on list so may be for that type only..

Message: Interface\AddOns\Carbonite.Quests\NxQuest.lua:4704: assertion failed!
Time: 05/06/17 11:49:06
Count: 6
Stack: [C]: in function `assert'
Interface\AddOns\Carbonite.Quests\NxQuest.lua:4704: in function `FindCur'
Interface\AddOns\Carbonite.Quests\NxQuest.lua:9700: in function `UserFunc'
Interface\AddOns\Carbonite\NxUI.lua:6500: in function `UserFunc'
Interface\AddOns\Carbonite\NxUI.lua:3680: in function <Interface\AddOns\Carbonite\NxUI.lua:3662>

Locals: (*temporary) = false

**Comment by IrcDirk** [comment]
Yep this assertion error is trown by Carbonite it self when QuestID is 0 or less. Maybe i will change this to something else in the future.

**Comment by Thrumbar** [comment]
This also relates to the above got this by clicking on the Scenario to track in Stockades..

18x Carbonite.Quests\NxQuest.lua:4704: assertion failed!
[C]: in function `assert'
Carbonite.Quests\NxQuest.lua:4704: in function `FindCur'
Carbonite.Quests\NxQuest.lua:9700: in function `UserFunc'
Carbonite\NxUI.lua:6499: in function `UserFunc'
Carbonite\NxUI.lua:3680: in function <Carbonite\NxUI.lua:3662>

Locals:
